flake8_plugins/assert_expected_actual.py

```diff
@@ -0,0 +1,107 @@
+import ast
+import json
+from pathlib import Path
+import re
+
+
+MSG = 'PTA100 expected should come before actual'
```

Comment:
is there a code convension for PTA100?

Reply:
I just followed [this video](https://youtu.be/ot5Z4KQPBL8) which said to use three letters followed by three numbers, so I used "PT" for "PyTorch", then "A" for "assert", then 100 because 100 seems like the simplest number

Reply:
I'm totally open to using a different code instead